deltachat-ios/Controller/WelcomeViewController.swift

```diff
@@ -90,6 +90,16 @@ class WelcomeViewController: UIViewController, ProgressAlertHandler {
         }
         navigationItem.rightBarButtonItem = moreButton
     }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        if let appDelegate = UIApplication.shared.delegate as? AppDelegate {
+            if !appDelegate.accountCreationQrCode.isEmpty {
```

Comment:
it would be nice if we could avoid keeping the state (`accountCreationQrCode`) directly in appDelegate.

Reply:
tried that first with passing it though the functions but it looked more annoying/complicated/too-verbose.
Is there another alternative that I don't know of yet?